src/sciline/data_graph.py

```diff
@@ -66,21 +67,39 @@ def __copy__(self: T) -> T:
         return self.copy()
 
     @property
-    def _graph(self) -> nx.DiGraph:
+    def index_names(self) -> tuple[IndexName, ...]:
+        """Names of the indices (dimensions) of the graph."""
+        return self._cbgraph.index_names
+
+    @property
+    def indices(self) -> dict[IndexName, Iterable[IndexValue]]:
+        """Names and values of the indices of the graph."""
+        return self._cbgraph.indices
+
+    @property
+    def cyclebane_graph(self) -> cb.Graph:
```

Comment:
Do we need this property? I don't see where it is used. And it adds one more concept to the interface. Users need to know what is the difference between all the different graphs.